docs/CONTRIBUTORS.rst

```diff
@@ -105,96 +35,77 @@ To work on the TUF project, it's best to perform a development install.
     $ python3 -m pip install -r requirements-dev.txt
 
 
-Auto-formatting
-===============
+Testing
+=======
 
-CI/CD will check that new TUF code is formatted with `black
-<https://black.readthedocs.io/>`__ and `isort <https://pycqa.github.io/isort>`__.
-Auto-formatting can be done on the command line:
-::
+With `tox <https://testrun.org/tox/>`_, the test suite can be executed in a
+separate *venv*. While a supported Python version must already be available,
+``tox`` will install ``tuf`` and dependencies, run the tests, auto-formatting,
+and coverage.
```

Comment:
I'm not a comma expert, but I think here's a superfluous one. Also, should we mention that tox runs tests for each Python version? E.g.:
```suggestion
With `tox <https://testrun.org/tox/>`_ the whole test suite can be executed in
a separate *virtual environment* for each supported Python version available on
the system. ``tuf`` and its dependencies are installed automatically for each tox
run.
```

Reply:
Thanks, I will look into the commas. Writing docs is a big challenge to me 🤪.

About the tox multiple versions, it doesn't run for all versions anymore in the default call. Not sure if we need to go deep on tox usage. The usage is in the link above.

Reply:
Oh right. Apologies for the noise then. Please disregard my suggestion.